.github/workflows/test-and-deploy.yml

```diff
@@ -331,12 +331,21 @@ jobs:
           if-no-files-found: error
           retention-days: 7
 
+      - name: Set asset name
+        run: |
+          if [[ ${{ matrix.runner }} == 'pi5-builder' ]]; then
```

Comment:
should use contain for safety this bash operation? `elif [[ ${{ matrix.os }} == *'pi5-builder'* ]];` then ?

Reply:
It's ok to use the machine name as it's.